commands/pipelines/setup.js

```diff
@@ -39,24 +40,33 @@ function createApp (heroku, { archiveURL, name, organization, pipeline, stage })
   })
 }
 
-function* getNameAndRepo (args) {
-  const answers = yield prompt([{
+function* getPipelineName (args) {
+  const answer = yield prompt([{
     type: 'input',
     name: 'name',
     message: 'Pipeline name',
     when () { return !args.name }
-  }, {
+  }])
+
+  const reply = Object.assign(answer, args)
+  reply.name = reply.name.toLowerCase().replace(/\s/g, '-')
+
+  return reply
+}
+
+function* getRepoName (args) {
```

Comment:
We can keep these together by using inquirer's `validate` functionality. This would also be nice because we can display a message without bombing out of the command